chart/values.yaml

```diff
@@ -0,0 +1,49 @@
+CliVersion: linkerd/cli edge-19.2.5
+ControllerComponentLabel: linkerd.io/control-plane-component
+ControllerImage: gcr.io/linkerd-io/controller:edge-19.2.5
+ControllerLogLevel: info
+ControllerReplicas: 1
+ControllerUID: 2103
+CreatedByAnnotation: linkerd.io/created-by
+DestinationAPIPort: 8086
+EnableH2Upgrade: true
+EnableHA: false
```

Comment:
It doesn't look like this is working as intended (replicas are staying at 1). Is that managed differently in the CLI?

Reply:
Looks like this flag has no effects on the replicas count in the [controller chart](https://github.com/linkerd/linkerd2/blob/d850b02b667dfdbbf08f4dcccf2046c5165d8209/chart/templates/controller.yaml#L102).

Reply:
Right, the CLI sets the controller's replicas programatically. This needs to be modified to be done in the chart directly instead. Something like: 
```
kind: Deployment
apiVersion: extensions/v1beta1
metadata:
  name: linkerd-controller
  namespace: {{.Values.Namespace}}
  labels:
    {{.Values.ControllerComponentLabel}}: controller
  annotations:
    {{.Values.CreatedByAnnotation}}: {{.Values.CliVersion}}
spec:
  replicas: {{- if .Values.EnableHA }}{{.Values.ControllerHAReplicas}}{{- else }}{{.Values.ControllerReplicas}}{{- end }}
```
where `ControllerHAReplicas` would be a new Value.

Reply:
I added this.